book/08-customizing-git/sections/attributes.asc

```diff
@@ -329,9 +329,9 @@ The substitutions can include for example the commit message and any `git notes`
 [source,console]
 ----
 $ echo '$Format:Last commit: %h by %aN at %cd%n%+w(76,6,9)%B$' > LAST_COMMIT
-$ git commit -am 'export-subst uses git log's custom formatter
+$ git commit -am 'export-subst uses git log'\''s custom formatter
```

Comment:
The commit message is on several lines. This is the whole commit message as it will appear:
```
export-subst uses git log's custom formatter

git archive uses git log's `pretty=format:` processor
directly, and strips the surrounding `$Format:` and `$`
markup from the output.
```
So nothing need to be changed, right ?